amqp/src/test/java/io/cloudevents/amqp/ProtonAmqpMessageWriterTest.java

```diff
@@ -21,43 +21,47 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Objects;
 
 import org.apache.qpid.proton.amqp.Binary;
 import org.apache.qpid.proton.amqp.messaging.ApplicationProperties;
 import org.apache.qpid.proton.amqp.messaging.Section;
 import org.apache.qpid.proton.message.Message;
-import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 
 import io.cloudevents.CloudEvent;
-import io.cloudevents.amqp.impl.ProtonBasedAmqpMessageWriter;
+import io.cloudevents.amqp.impl.ProtonAmqpMessageWriter;
 import io.cloudevents.core.format.EventFormat;
 import io.cloudevents.core.message.MessageWriter;
 import io.cloudevents.core.mock.CSVFormat;
-import io.cloudevents.core.test.Data;
-import io.cloudevents.types.Time;
 
 /**
- * Tests verifying the behavior of the {@code ProtonBasedAmqpMessageWriter}.
+ * Tests verifying the behavior of the {@code ProtonAmqpMessageWriter}.
  */
-public class ProtonBasedAmqpMessageWriterTest {
+public class ProtonAmqpMessageWriterTest {
 
     /**
      * Verifies that a binary CloudEvent message can be successfully represented
      * as an AMQP message.
      */
-    @Test
-    public void testWriteBinaryCloudEventToAmqpRepresentation() {
-
-        final CloudEvent binaryEvent = Data.V1_WITH_JSON_DATA;
-        final Message expectedMessage = translateBinaryEvent(binaryEvent);
+    @ParameterizedTest()
+    @MethodSource("io.cloudevents.core.test.Data#allEventsWithStringExtensions")
+    public void testWriteBinaryCloudEventToAmqpRepresentation(final CloudEvent binaryEvent) {
+
+        Message expectedMessage = null;
+        try {
+            expectedMessage = translateBinaryEvent(binaryEvent);
+        } catch (final Exception e) {
+            System.out.println(e);
```

Comment:
i used this for debugging but forgot to remove. will include this in the final update